src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.EasyRequest.cs

```diff
@@ -38,6 +44,12 @@ private sealed class EasyRequest : TaskCompletionSource<HttpResponseMessage>
             internal SendTransferState _sendTransferState;
             internal bool _isRedirect = false;
 
+            private SafeCallbackHandle _headerCallbackHandle;
+            private SafeCallbackHandle _sendCallbackHandle;
+            private SafeCallbackHandle _receiveBodyCallback;
+            private SafeCallbackHandle _seekCallbackHandle;
+            private SafeCallbackHandle _sslCtxCallbackHandle;
```

Comment:
This is now increasing the size of the EasyRequest object by five reference fields, and most or all requests are incurring five more allocated finalizable SafeHandles.  Is there anything we can do to avoid that?  At a minimum could we combine them all into a single SafeCallbacksHandle?  Though it'd be nice if we could avoid this entirely.